client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -448,7 +494,11 @@ def fetch_artifacts(self, keys):
             #     identical, so multiple writes would be idempotent.
             path = self._get_cached_file(filename)
             if path is None:
-                contents = self._get_artifact(key)
+                try:
```

Comment:
A `ValueError` is raised by `_get_artifact` if the artifact is incomplete. I have guessed that partial success is a better behavior for this method than total failure if any one of the artifacts has an incomplete upload.